Rubberduck.VBEEditor/Events/AutoCompleteEventArgs.cs

```diff
@@ -1,57 +1,64 @@
 ﻿using System;
+using System.Windows.Forms;
 using Rubberduck.VBEditor.SafeComWrappers.Abstract;
 
 namespace Rubberduck.VBEditor.Events
 {
     public class AutoCompleteEventArgs : EventArgs
     {
-        public AutoCompleteEventArgs(ICodePane pane)
+        public AutoCompleteEventArgs(ICodeModule module, WindowsApi.KeyPressEventArgs e)
         {
-            CodePane = pane;
-            var selection = pane.Selection;
-            using (var module = pane.CodeModule)
+            if (e.Key == Keys.Delete ||
+                e.Key == Keys.Back ||
+                e.Key == Keys.Enter ||
+                e.Key == Keys.Tab)
```

Comment:
maybe extract this into something like `IsRelevantNonPrintable` or similar. Also note that you don't handle a TON of other non-printables